cmake/ucx-config.cmake.in

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (C) NVIDIA Corporation. 2021.  ALL RIGHTS RESERVED.
+#
+
+set(prefix "@prefix@")
+set(exec_prefix "@exec_prefix@")
```

Comment:
since we include ucx-targets.cmake, can avoid re-defining prefix and exec_prefix?
BTW, why need to separate to 2 cmake files (ucx-config and ucx-targets)?

Reply:
I have updated this PR to avoid redefinition. There are two separate files because other projects do so. I guess this is how cmake generates these files (See https://cmake.org/cmake/help/git-stage/guide/importing-exporting/index.html#exporting-targets-from-the-build-tree). But I don't know if this separation is required or not.